discoverygarden/islandora_solr
# Removing search breadcrumbs and fixing a minor config bug

# Removing Search Breadcrumbs

# What does this Pull Request do?

Removes search breadcrumbs. Ensures `islandora_solr_breadcrumbs_parent_fields` is respected.

The functionality is redundant with the query block and the use case is not directly supported by the D8 breadcrumb system. So we are removing rather than forcing this unintended design on the breadcrumb system. 

# How should this be tested?

In the breadcrumb admin change the parent fields and verify object breadcrumbs respect it.
Note that the config for `islandora_solr_breadcrumbs_add_collection_query` is no longer present.

# Additional Notes:

* Change notes in the D8 Islandora wiki should be updated.
* Similar UX can be achieved through styling the query block.
